shared/js/views.jsx

```diff
@@ -831,6 +927,32 @@ loop.shared.views = (function(_, mozL10n) {
     }
   });
 
+  var RemoteCursorView = React.createClass({
+    mixins: [React.addons.PureRenderMixin],
+
+    propTypes: {
+      offsetLeft: React.PropTypes.number,
+      offsetTop: React.PropTypes.number,
+      remoteCursorLeft: React.PropTypes.number,
+      remoteCursorTop: React.PropTypes.number
+    },
+
+    render: function () {
+      console.log("remoteCursorTop", this.props.remoteCursorTop);
+      console.log("remoteCursorLeft", this.props.remoteCursorLeft);
+      console.log("offsetTop", this.props.offsetTop);
+      console.log("offsetLeft", this.props.offsetLeft);
+      var cursorStyle = {
```

Comment:
Yeah, this variable defines the cursor position but i keep that name because we're setting the style of the element